string author
string receiver
string talk

